src/index.js

```diff
@@ -8,21 +15,50 @@ const removeLeadingDotSlash = path => (path && path.match(/^\.\//)
   ? path.replace(/^\.\//, '')
   : path
 );
+const buildPluginConfig = opts => ({
+  dropAllFilenames: opts && opts.dropAllFilenames,
+  dropExtensions: (opts && opts.dropExtensions) || defaultDropExtensions,
+  separator: opts && opts.separator,
+  userSetRoot: opts && removeLeadingDotSlash(normalisePathSep(opts.root)),
+});
+const getConfigIfNodeTransformable = (path, state) => {
+  // Node matches default placeholder (added in v1).
+  if (path.node.name === '__filenamespace') {
+    return buildPluginConfig(state.opts);
+  }
+  // Node matches custom placeholder (added in v2).
+  const placeholder = (state.opts.customFilenamespacePlaceholders || ['.spec', '.test'])
```

Comment:
Looks like some testing code left behind, assume `|| ['.spec', '.test']` should be `|| []`???